src/braket/circuits/angled_gate.py

```diff
@@ -46,7 +46,7 @@ def __init__(
                 printing a diagram of a circuit. The length must be the same as `qubit_count`, and
                 index ordering is expected to correlate with the target ordering on the instruction.
                 For instance, if a CNOT instruction has the control qubit on the first index and
-                target qubit on the second index, the ASCII symbols should have `["C", "X"]` to
+                target qubit on the second index, the ASCII symbols should have `["⏺", "X"]` to
```

Comment:
I believe this is the wrong character being used. On mobile it shows up as ⏺️. I believe ● is the correct character.

![image](https://github.com/amazon-braket/amazon-braket-sdk-python/assets/20567262/c6897519-940c-4f48-9c2e-acb4a5ee8e39)